### ISSUE TITLE
a11y: Fix aria-label, role and focus display for head button icons

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12792]
a11y Issue: 26

For the header menu bar icons (see below).

The mentioned image buttons lack textual description resulting in screen reader users found it difficult to identify the purpose of these buttons, e.g.

- Import YAML
- Download KubeConfig
- Copy KubeConfig to Clipboard and so on image buttons

Ensure that appropriate textual description such as "Import YAML", "Download KubeConfig", "Copy KubeConfig to Clipboard" and so on is provided for the image buttons using "aria-label" attribute.

![image](https://github.com/user-attachments/assets/7a9acf6c-81fb-4cbd-9e0d-dcf2e2604711)

**Comment by nwmac** [comment]
Duplicate
